django_project/lesson/utilities.py

```diff
@@ -60,3 +64,23 @@ def re_order_features(request, features):
             feature.save()
 
     return HttpResponse('')
+
+
+def validate_zipfile(file) -> bool:
+    try:
+        zip = zipfile.ZipFile(file)
+    except:
+        raise ValidationError(_("Could not unzip file."))
+    for zname in zip.namelist():
+        if zname.find('..') != -1 or zname.find(os.path.sep) == 0:
+            raise ValidationError(
+                _('For security reasons, zip file cannot contain path '
+                  'informations')
+            )
+        for forbidden_dir in ['__MACOSX', '.git', '__pycache__']:
```

Comment:
@sumandari it would be nice if we put this list in something like an ignore list file.